src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -747,6 +803,33 @@ def _on_acquire_overview(self, _):
         if das:
             self._tab.load_overview_data(das)
 
+    def _on_tdct(self, _):
+        """
+        called when the button "TDCT" is pressed
+        """
+        for stream in self._tab_data.main.currentFeature.value.streams.value:
```

Comment:
Is there a need to check if current feature is not None?

Reply:
It will not be None as check_correlation_conditions checks that there is always a current feature and other conditions before enabling the correlation button.